pkg/api/api.go

```diff
@@ -258,7 +260,11 @@ func (a API) ReceiveEvent(w http.ResponseWriter, r *http.Request) {
 				)
 			defer span.End()
 
-			id, err := a.handler(ctx, &evt)
+			seed := event.SeededIDFromString(
+				r.Header.Get("x-inngest-idempotency-key"),
```

Comment:
(Discussed - using as a reminder)
```suggestion
				r.Header.Get("x-inngest-event-id-seed"),
```
Nit: Could this also be a const somewhere?